### PR TITLE
Added the MongoDB 3 repo to the whitelist

### PR DESCRIPTION
**Pull request by thedrow**
This is required for MongoEngine in order to move to the new infrastrucutre.

### CODE REVIEW

File: ubuntu.json [hunk 1]
```diff
@@ -110,6 +110,12 @@
     "key_url": "http://keyserver.ubuntu.com/pks/lookup?op=get&fingerprint=on&search=0x9ECBEC467F0CEB10"
   },
   {
+    "alias": "mongodb-precise",
+    "sourceline": "deb http://repo.mongodb.org/apt/ubuntu precise/mongodb-org/3.0 multiverse",
+    "key_url": "http://keyserver.ubuntu.com/pks/lookup?op=get&fingerprint=on&search=0x9ECBEC467F0CEB10"
+  },
+  }
```
Comment (stoshiya) [inline ubuntu.json]: Remove this closing bracket.

**Comment by owenallenaz** [comment]
Hey @thedrow I was looking for this same feature, the Travis build is failing because they require the aliases to be sorted alphabetically. Could you resort the mongodb-precise before the mongodb-upstart and re-submit.

**Comment by thedrow** [comment]
@owenallenaz done!

**Comment by owenallenaz** [comment]
Hmm... well then, still not passing. The only thing I can think of is possibly the extra space before the `{`, the error message in the build is sort of cryptic.

**Comment by thedrow** [comment]
I removed the extra space. Let's see what the build says.

**Comment by thedrow** [comment]
Yeh that was it. @BanzaiMan Can you please merge this so we can whitelist the mongodb 3.x packages?

**Comment by psi29a** [comment]
Please merge this, I need this for our TxMongo builds:
https://travis-ci.org/twisted/txmongo
https://github.com/twisted/txmongo

Thanks! :)

**Comment by BanzaiMan** [comment]
I've renamed the source to "mongodb-3.0-precise".

**Comment by psi29a** [comment]
Cheers! :)

**Comment by psi29a** [comment]
When will this go live? It is currently sitting in patch-1 and we need it ASAP. :)

**Pull request merged**